offchain/webpack.config.js

```diff
@@ -76,11 +76,6 @@ module.exports = {
     new webpack.LoaderOptionsPlugin({
       debug: true,
     }),
-    new HtmlWebpackPlugin({
-      title: "hello-world-api",
-      template: "./index.html",
-      inject: false, // See stackoverflow.com/a/38292765/3067181
-    }),
```

Comment:
Why was this deleted? this `webpack.config.js` was copied from https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/webpack.config.js

Reply:
yes, it was copied from CTL then I found the stanza is useless since we're not hosting an index page as CTL does